services/admin.ts

```diff
@@ -277,10 +299,10 @@ export class AdminZkRandService extends BaseService<AdminZkRandOptions> {
         await waitForWriteJsonToFile(obj, instancesPath)
         console.log("retrieved all instances from contract")
         console.log("sleeping..")
-        await sleep(2000)
+        await sleep(5000)
```

Comment:
Can we not wait for this asynchronously?

Reply:
writing to disk takes some time to finish. The command after will fail if no wait or not enough delay